### DISCUSSION TITLE
Foundry Fundamentals, Section 2,Lesson 9

### DISCUSSION THREAD

**Discussion by Donpele01** [Cyfrin/foundry-full-course-cu#1984] [Q&A] [answered]
Please i am having this error when running this command:
''' forge test --fork-url $SEPOLIA_RPC_URL '''

Here is the error
...
solidity
Compiler run failed:
Error (9553): Invalid type for argument in function call. Invalid implicit conversion from contract AggregatorV3Interface to contract AggregatorV3Interface requested.
  --> src/FundMe.sol:27:45:
   |
27 |         require(msg.value.getConversionRate(s_priceFeed) >= MINIMUM_USD, "You need to spend more ETH!");
   |                                             ^^^^^^^^^^^
...

**Comment by EngrPips** [comment, marked as answer]
Hello @Donpele01, This discussion `https://github.com/Cyfrin/foundry-full-course-cu/discussions/1948#discussion-6902122` treats the same issue you are running into. Please go through and let us know if you are able to resolve your issue.

**Comment by Donpele01** [comment]
Thanks, i have resolved it.

**Reply by EngrPips** [reply]
Awesome, I am glad that helped. Keep crushing it.
